src/components/Filters/Filters.stories.tsx

```diff
@@ -18,7 +18,7 @@ import {
   Filters,
   GridColumn,
   GridDataRow,
-  GridTable,
+  GridTableUnsafe as GridTable,
```

Comment:
@bdow I'd added `GridTableUnsafe` as an alias with the old/as-is props, but kinda wondering if maybe we should take this opportunity to introduce `Table` as the new component name, which has the new/safe props, and `GridTable` is the old/deprecated component name.

That would make this roll out a lot easier, b/c all exists clients of `GridTable` wouldn't have to be either a) renamed over to the `GridTableUnsafe` alias, or b) have their non-memoed rows/columns fixed.

...although arguably b) would be a great forcing function, it just means rolling this out will be a lot more work/updates.

Wdyt?

Reply:
Yea, I like that idea to rename the "new" to just `Table`. Then we won't block updating Beam on updating existing GridTable implementations.

Reply:
Cool! Pushed up the "GridTable stays the same, Table is the new safe one".